src/pytest_dbt_core/fixtures.py

```diff
@@ -58,6 +59,12 @@ def config(request: SubRequest) -> RuntimeConfig:
     RuntimeConfig
         The runtime config.
     """
+
+    # only override profiles_dir if its set, otherwise keep DBT behavior
+    profiles_dir = request.config.getoption("--profiles-dir")
+    if profiles_dir:
+        flags.PROFILES_DIR = profiles_dir
```

Comment:
Nicely done! I trust you found that this is the dbt way of setting the profiles directory?

Reply:
Yep, you can see them doing it [here](https://github.com/dbt-labs/dbt-core/blob/c82795a99f13bcbe0a86e6084432702bbef214eb/core/dbt/main.py#L1225)

Reply:
Check, could you add an `abspath` like they do [there](https://github.com/dbt-labs/dbt-core/blob/c82795a99f13bcbe0a86e6084432702bbef214eb/core/dbt/main.py#L1222)?

And I am a bit confused about [dbt removing their main](https://github.com/dbt-labs/dbt-core/commit/c53c3cf181bf4facfb34ea5068d033bfc314eb72), where this piece of code is coming from. But if this works for now, let's merge this and tackle that later (if it is required).

Reply:
sure thing: https://github.com/godatadriven/pytest-dbt-core/pull/25/commits/1ddb9791e1ddfb1a157dd31d70d7d02c9610ec68

it looks like `1.5` re-arranged a bunch of code (like you mention, e.g. removing main) - I don't know the full extent of those changes, but I'm suspecting it'll involve a bit of work within this library to support it. they've got a release-candidate out so we could start poking around

Reply:
You're very much welcome to have a look around! And let's merge this PR first.